oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/searchtag/TagAutocompleteDispatcher.java

```diff
@@ -30,7 +30,8 @@ public void dispatch(TagAutocomplete source) {
         autocomplete.setTagKey(source.getTagKey());
         autocomplete.setTagValue(source.getTagValue());
         autocomplete.setTagType(source.getTagType().name());
-        autocomplete.setTimeBucket(source.getTimeBucket());
+        // change the precision in Day for reduce the storage
+        autocomplete.setTimeBucket(source.getTimeBucket() / 10000 * 10000);
```

Comment:
`TimeBucket#retainToDay4Minute` should be the place to host this `/10000 * 10000` logic